packages/backend/src/services/GithubAppService/GithubAppService.ts

```diff
@@ -111,6 +101,57 @@ export class GithubAppService extends BaseService {
             if (refreshToken === undefined)
                 throw new ForbiddenError('Invalid authentication token');
 
+            const redirectUrl = new URL(oauth?.returnTo || '/');
+
+            if (setup_action === 'request') {
+                // User attempted to setup the app, didn't have permission in GitHub and sent a request to the admins
+                // We will try to poll for the installation id
+                console.info(
+                    'Waiting for Github app to be authorized by an admin',
+                );
+
+                const interval = setInterval(async () => {
```

Comment:
Please read this https://github.com/lightdash/lightdash/issues/11987#issuecomment-2425893582